sdk/identity/Cargo.toml

```diff
@@ -50,3 +53,7 @@ enable_reqwest_rustls = [
 ]
 development = []
 test_e2e = []
+
+[[example]]
+name="client_certificate_credentials"
+required-features = ["openssl"]
```

Comment:
Perhaps make this a named feature such as `client_certificate` rather than `openssl`?

Reply:
Resolved. Made it an optional feature. I was debating about it but initially opted not to.

Reply:
Could you rename the feature to `client_certificate` instead of `openssl`?

Reply:
done